config.c

```diff
@@ -1071,7 +1071,7 @@ static int readConfigFile(const char *configFile, struct logInfo *defConfig)
                     "Ignoring %s because the file owner is wrong (should be root or user with uid 0).\n",
                     configFile);
             close(fd);
-            return 0;
+            return 1;
```

Comment:
I think we should equally handle the `(sb_config.st_mode & 0022)` condition above.

Reply:
I added it in new commit (https://github.com/logrotate/logrotate/pull/508/commits/86bffca5637a699354cb13d9d5cc7beb2d245910).